backend/server.js

```diff
@@ -1352,10 +1369,76 @@ router.post('/run-clubs/:clubId/messages', authenticateToken, async (req, res) =
     });
     res.status(201).json(message);
   } catch (error) {
-    console.error('Error sending message:', error);
     res.status(500).json({ error: 'Failed to send message', details: error.message });
   }
 });
 
+// Fetch club members
+app.get('/run-clubs/:clubId/members', authenticateToken, async (req, res) => {
+  try {
+    const members = await prisma.clubMember.findMany({
+      where: { clubId: parseInt(req.params.clubId) },
+      include: { User: { select: { username: true } } },
+    });
+    res.json(members.map(member => member.User));
+  } catch (error) {
+    res.status(500).json({ error: 'Failed to fetch club members', details: error.message });
+  }
+});
+
+// Fetch club statistics
+app.get('/run-clubs/:clubId/statistics', authenticateToken, async (req, res) => {
+  try {
+    const clubMembers = await prisma.clubMember.findMany({
+      where: { clubId: parseInt(req.params.clubId) },
+      select: { userId: true },
+    });
+    const memberIds = clubMembers.map(member => member.userId);
+
+    const activities = await prisma.userActivity.findMany({
+      where: { userId: { in: memberIds } },
+    });
+
+    const totalDistance = activities.reduce((sum, activity) => sum + activity.distance, 0);
+    const totalDuration = activities.reduce((sum, activity) => sum + activity.duration, 0);
+    const averagePace = totalDistance > 0 ? (totalDuration / 60) / totalDistance : 0;
```

Comment:
nit: make `60` a named constant instead of having it be an inline magic number

Reply:
Addressed the nit and made a variable for it